translations/en.json

```diff
@@ -11,6 +11,8 @@
   "closesNotification": "Closes notification",
   "createdRecord": "Record created",
   "createdRecordDescription": "A new encounter was created",
```

Comment:
Any ideas why these keys didn't get removed?

Reply:
Thought they're being used somewhere else, is it okay if I remove them?!

Reply:
AFAICT, they're only used in the spot we fixed... If they were used somewhere else, that's somewhere else we should fix.